crates/turborepo-auth/src/auth/login.rs

```diff
@@ -79,201 +79,14 @@ pub async fn login<'a>(
 
 #[cfg(test)]
 mod tests {
-    use std::sync::atomic::AtomicUsize;
```

Comment:
To save whoever looks at this some time: I moved all this stuff into `turborepo-auth/src/mocks`